api/app/tests/apps/api/v1/test_private_signal_endpoint.py

```diff
@@ -1549,6 +1549,22 @@ def test_create_initial_email_phone_empty_string(self, validate_address_dict):
         self.assertEqual(new_signal.reporter.email, None)
         self.assertEqual(new_signal.reporter.phone, None)
 
+    def test_detail_endpoint_SIG_2486(self):
+        """
+        The date in the _display value did showed with the correct timestamp as all other dates
+        """
+        response = self.client.get(self.detail_endpoint.format(pk=self.signal_no_image.id))
+        self.assertEqual(response.status_code, 200)
+
+        # JSONSchema validation
+        data = response.json()
+        self.assertJsonSchema(self.retrieve_signal_schema, data)
+
+        created_at = self.signal_no_image.created_at.astimezone(timezone.get_current_timezone())
+        created_at_with_time_zone_str = created_at.isoformat()
+        self.assertIn(created_at_with_time_zone_str, data['_display'])
+        self.assertEqual(created_at_with_time_zone_str, data['created_at'])
```

Comment:
Possibly add a test checking `updated_at` as well.

Reply:
Will do this in another ticket where we want to refactor the signals test suite